src/test/java/net/library/controller/BookControllerTest.java

```diff
@@ -21,23 +23,23 @@
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 
 import java.time.LocalDateTime;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
 import java.util.concurrent.Callable;
 
 import static net.library.tools.Tools.*;
 import static net.library.util.HttpUtil.*;
 import static net.library.util.Utils.getUUID;
 import static org.hamcrest.Matchers.*;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
```

Comment:
should be replaced with a call to a method from 'org.junit.jupiter.api.Assertions'